x/ibc/04-channel/keeper/timeout.go

```diff
@@ -122,17 +124,15 @@ func (k Keeper) TimeoutPacket(
 }
 
 // TimeoutExecuted deletes the commitment send from this chain after it verifies timeout
-func (k Keeper) TimeoutExecuted(ctx sdk.Context, packet exported.PacketI) error {
+func (k Keeper) TimeoutExecuted(ctx sdk.Context, chanCap *capability.Capability, packet exported.PacketI) error {
```

Comment:
I noticed TimeoutExecuted needs a capability, does `PacketExecuted` also need a capabality as well?

Reply:
@cwgoes ?

Reply:
Ah yes, it ought to authenticate, good catch.